src/router/index.js

```diff
@@ -0,0 +1,39 @@
+import { createRouter, createWebHistory } from 'vue-router';
+
+const routes = [
+    {
+        path: '/',
+        name: 'homepage',
+        component: function () {
+            return import('../views/Homepage.vue');
+        },
+    },
+    {
+        path: '/legal/terms',
+        name: 'terms',
+        component: function () {
+            return import('../views/legal/Terms.vue');
+        },
+    },
+    {
+        path: '/legal/privacy',
+        name: 'terms',
```

Comment:
```suggestion
        name: 'privacy',
```
should we use privacy here?

Reply:
have fixed it